indexes/plugin.js

```diff
@@ -44,17 +44,17 @@ module.exports = function (
       writeData((err) => {
         if (err) return cb(err)
         else {
-          seq.set(unWrittenSeq)
+          offset.set(unWrittenOffset)
           cb()
         }
       })
     } else cb()
   }
 
-  function onData(data, isLive) {
+  function onData(record, isLive) {
     // maybe check if for us!
-    let unwritten = handleData(data, processed)
-    if (unwritten > 0) unWrittenSeq = data.seq
+    let unwritten = handleData(record, processed)
+    if (unwritten > 0) unWrittenOffset = record.seq
```

Comment:
this should be offset

Reply:
Ah right, a bit confusing

Reply:
Pushed a comment instead, I wonder if we should land a change to async-flumelog before merging this. It would cut down some of the confusion and it seems to be quite a small change. What do you think @staltz?

Reply:
Sounds good by me. Actually async-flumelog is a bit of a wildcard in the flumedb family. It's not quite conforming to flumedb, which would require the name seq. I'd be more comfortable if async-flumelog wouldn't have flume in the name, but even if it has flume in the name, I'm fine with renaming seq to offset there

Reply:
Yeah, one could always do a wrapper to make it conform to flume seq again. I think I'll do that (figure out a good name) and move the repo to ssb-ngi-pointer then.

Reply:
Nice to move it to this GH org :+1: